PrestaShop/PrestaShop
# Reorganize default configuration

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Cleans up the default configuration file. All configuration keys are now groupped and sorted by how they are used in backoffice. Then I separated entries that mark order statuses, entries that mark movement reasons, entries that are internal only, entries that are module specific but in a core and finally entries that not used anywhere and should be removed in v9. I also added comments with missing configuration entries.
| Type?             | refacto
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | See below
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   | 

### How to test
Probably to be tested by a dev.

1. Install clean 8.1.x
2. Go to phpmyadmin, open ps_configuration, show all entries, sort it by `name`, select all lines, click Export and confirm.
3. Install this PR to another folder.
4. Go to phpmyadmin and export the data again.
5. Compare the two files (https://www.diffchecker.com/) and check that only times and maybe some random strings have changed.
